LayoutTests/fast/dom/timer-in-microtask-does-not-inherit-nesting-level.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../resources/js-test.js"></script>
+<script>
+jsTestIsAsync = true;
+description("Tests that a timer that is scheduled during a microtask checkpoint does not inherit the nesting level of the timer task that just finished running.");
```

Comment:
It would be nice to write a test which checks the throttling of the timers directly.
That way, we can confirm this fix is working within a browser instead of having to use WKTR.